bin/tic_tac_toe.rb

```diff
@@ -6,12 +6,12 @@
 
 include SettingTypes
 
-game_settings = GameSettingSetter.new()
+game_settings = GameSettingSetter.new
 settings = game_settings.create_settings
 
 game = Game.new(
-    board_size: settings[BOARD_SIZE],
-    player_2: settings[OPPONENT_TYPE_SETTING],
+    board_size: settings[SettingTypes::BOARD_SIZE],
+    player_2: settings[SettingTypes::OPPONENT_TYPE_SETTING],
```

Comment:
it's a lot clearer now that `player_2` refers to the opponent now. I have a few questions - does `player_2` always go second or is this just considered the other player? Does this still work if I asked you to support allowing a computer to play against a computer? Why does the game constructor only require configuration for `player_2` - is it because `player_1` is always human?

Reply:
player_2 always go second, and game constructor only exposes player_2 as a setting option because the game feature only had humans as the first player thus far. I had both player_1 and player_2 exposed before, but since it was always human first, I just removed it and used a default value.

If computer vs computer was an option, it's enough to adjust.  I would just add a player_1 player type setting, then expose the player_1 configuration as well.